src/main/java/de/bonndan/nivio/notification/MessagingService.java

```diff
@@ -29,11 +31,25 @@ public MessagingService(SimpMessagingTemplate template) {
         this.template = template;
     }
 
-    @Override
-    public void onApplicationEvent(ProcessingEvent processingEvent) {
+    @EventListener(ProcessingFinishedEvent.class)
+    public void onProcessingEvent(ProcessingEvent processingEvent) {
         EventNotification eventNotification = EventNotification.from(processingEvent);
         fifo.add(eventNotification);
-        LOGGER.info("Broadcasting processing event: " + processingEvent.getType());
+        LOGGER.debug("Broadcasting processing event: " + processingEvent.getType());
+        this.template.convertAndSend(WebSocketConfig.TOPIC + EVENTS, eventNotification);
+    }
+
+    @EventListener(InputChangedEvent.class)
```

Comment:
I couldn't find who triggered this event, then I saw the changes in `LocalFileObserver`, but these are not in this branch. I guess the commit to develop was not on purpose?

Anyhow, didn't we send an event via websockets on `InputChangedEvent` before? Only on `ProcessingFinishedEvent`? Does this change anything in the notification of the frontend?

Reply:
As I have seen so far, we only use `ProcessingFinishedEvent` in the frontend. What is `InputChangedEvent` for, especially sending it via websockets to the frontend?

Reply:
InputChangedEvent is fired when files change, "long" before the processing has finished. It would notify the user that something will happen behind the curtains now. However, I will recheck that behavior again.